From: Takashi Iwai <tiwai@suse.de>
Subject: kABI workaround for usb/host/xhci-hub.c
Patch-mainline: Never, kABI workaround
References: git-fixes

The patch
  patches.suse/xhci-Restrict-USB4-tunnel-detection-for-USB3-devices.patch
added the inclusion of linux/pci.h, which broke kABI due as the defined
structs.  Simply wrap it for kABI compatibility.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/usb/host/xhci-hub.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/usb/host/xhci-hub.c
+++ b/drivers/usb/host/xhci-hub.c
@@ -12,7 +12,9 @@
 #include <linux/slab.h>
 #include <asm/unaligned.h>
 #include <linux/bitfield.h>
+#ifndef __GENKSYMS__
 #include <linux/pci.h>
+#endif
 
 #include "xhci.h"
 #include "xhci-trace.h"
